Berlin/PetermannIINachtrag42.xml

```diff
@@ -29,17 +26,119 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+                     </note>
+                     <note>The following beginnings are rubricated, but not indicated in the margin:
+                     Jer 11:1, 13:8, 14:1, 15:1, 18:1, 25:1 (wrongly indicated as 25:14 in the text), 26:1, 26:7, 28:12, 30:1, 44:15, 46:1, 47:1, 46:13 (as 46:13-24 is located between 47:3 and 4),
+                     48:1, 48:36, 49:1, 49:7, 49:28, 49:34, 50:1, 52:6, 52:12.</note>
+                     <note>The titles <foreign xml:lang="gez">ዳግም፡</foreign>, <foreign xml:lang="gez">ሣልስ፡</foreign> and
+                        <foreign xml:lang="gez">ራብዕ፡ ዜማ፡ ዘእምሰቆቃው፡</foreign> are indicated in red in the upper margin of the second, third and fourth lamentations.</note>
+                     
+                     <msItem xml:id="ms_i2.1">
+                        <locus from="93ra" to="196vb"/>
+                        <title type="complete" ref="LIT1685Bookof"/>
+                        <textLang mainLang="gez"/>
+                        
+                        <msItem xml:id="ms_i2.1.1">
+                           <locus from="131vb" to="132ra"/>
+                           <title type="complete" ref="LIT2193Prophe"/>
+                           <textLang mainLang="gez"/>
+                           <note>The text of the Prophecy of Jeremiah Told to Pashhur is part of the Book of Jeremiah between 
+                           Jer. 20:3-4.</note>
+                        </msItem>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i2.2">
+                        <locus from="196vb" to="200va"/>
+                        <title type="complete" ref="LIT1202Bookof"/>
+                        <textLang mainLang="gez"/>
+                        <note>This work is not set apart codicologically from the preceding <ref target="#ms_i2.1"/>.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i2.3">
+                        <locus from="200va" to="206va"/>
+                        <title type="complete" ref="LIT1753Lament"/>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                  
+                     <msItem xml:id="ms_i2.4">
+                        <locus from="206va" to="207vb"/>
+                        <title type="complete" ref="LIT1686Epistl"/>
+                        <textLang mainLang="gez"/>
+                        <explicit type="subscription" xml:lang="gez">ተፈጸመ፡ መጽሐፈ፡ እርምያስ፡ ነቢይ፡</explicit>
+                     </msItem>
+                   
+                     <msItem xml:id="ms_i2.5">
+                        <locus from="208ra" to="219rb"/>
+                        <title type="complete" ref="LIT2167Parali"/>
+                        <textLang mainLang="gez"/>
+                        <explicit type="subscription" xml:lang="gez">ወትሩፋተ፡ ነገሩሰ፡ ለእርምያስ፡ ነዋ፡ ጽሑፍ፡ ውስተ፡ መጽሐፈ፡ ባሮክ። ወተፈጸመ፡ ዘባሮክ።</explicit>
```

Comment:
The manuscript seems to have ትራፋተ፡. Also, do we not use " ፨" for the nine-dot asterisk?